keyboards/crkbd/keymaps/nimishgautam/README.md

```diff
@@ -0,0 +1,47 @@
+# Full-featured Corne layout for Linux and MacOS
```

Comment:
Please rename this file to be all lower case.

Reply:
done

Reply:
Sadly, nope.  

Using the web editor may help.  Otherwise, name it to something else, commit, rename to `readme.md` and commit again.

Reply:
Gah. Done now when I look everywhere. Thanks for the patience!